Importer/Interpreter/EntityDataInterpreterFactory.php

```diff
@@ -43,11 +44,14 @@ public function __construct(
 
     /**
      * @param EntityConfigurationInterface $configuration
+     * @param LogInterface                 $log
      *
      * @return EntityDataInterpreter
      */
-    public function create(EntityConfigurationInterface $configuration)
+    public function create(EntityConfigurationInterface $configuration, LogInterface $log)
     {
-        return new EntityDataInterpreter($configuration, $this->entityManager, $this->validator, $this->eventDispatcher);
+        $interpreterExceptionEventFactory = new InterpreterExceptionEventFactory($log);
```

Comment:
This looks a bit weird, creating new factory for each interpreter.
Maybe moving log here wasn't the best idea.

But this intermediate step can be improved later.